packages/playground/src/components/Playground.tsx

```diff
@@ -0,0 +1,400 @@
+import { useCallback, useState, useRef, useEffect, type ComponentType, type FormEvent } from 'react';
+import 'react-app-polyfill/ie11';
+import isEqualWith from 'lodash/isEqualWith';
+import Form, { withTheme, type IChangeEvent, type FormProps } from '@rjsf/core';
+import {
+  type ErrorSchema,
+  type TemplatesType,
+  type ArrayFieldTemplateProps,
+  type ObjectFieldTemplateProps,
+  type RJSFSchema,
+} from '@rjsf/utils';
+import localValidator from '@rjsf/validator-ajv8';
+
+import { samples } from '../samples';
+import DemoFrame from './DemoFrame';
+import ErrorBoundary from './ErrorBoundary';
+import CopyLink from './CopyLink';
+import GeoPosition from './GeoPosition';
+import ThemeSelector from './ThemeSelector';
+import Selector from './Selector';
+import ValidatorSelector from './ValidatorSelector';
+import SubthemeSelector from './SubthemeSelector';
+import RawValidatorTest from './RawValidatorTest';
+import Editor from './Editor';
+
+const log = (type: string) => console.log.bind(console, type);
+const toJson = (val: unknown) => JSON.stringify(val, null, 2);
+
+const liveSettingsSchema = {
+  type: 'object',
+  properties: {
+    liveValidate: { type: 'boolean', title: 'Live validation' },
+    disable: { type: 'boolean', title: 'Disable whole form' },
+    readonly: { type: 'boolean', title: 'Readonly whole form' },
+    omitExtraData: { type: 'boolean', title: 'Omit extra data' },
+    liveOmit: { type: 'boolean', title: 'Live omit' },
+    noValidate: { type: 'boolean', title: 'Disable validation' },
+    focusOnFirstError: { type: 'boolean', title: 'Focus on 1st Error' },
+    showErrorList: {
+      type: 'string',
+      default: 'top',
+      title: 'Show Error List',
+      enum: [false, 'top', 'bottom'],
+    },
+  },
+};
+
+type EditorsProps = {
+  schema: unknown;
+  setSchema: React.Dispatch<React.SetStateAction<object>>;
+  uiSchema: unknown;
+  setUiSchema: React.Dispatch<React.SetStateAction<object>>;
+  formData: unknown;
+  setFormData: React.Dispatch<React.SetStateAction<object>>;
+  extraErrors: unknown;
```

Comment:
Consider using the types:
```suggestion
  schema: RJSFSchema;
  setSchema: React.Dispatch<React.SetStateAction<object>>;
  uiSchema: UiSchema;
  setUiSchema: React.Dispatch<React.SetStateAction<object>>;
  formData: any;
  setFormData: React.Dispatch<React.SetStateAction<object>>;
  extraErrors: ErrorSchema;
```